README.md

```diff
@@ -69,7 +69,7 @@ forcefield = ForceField('Frosst_AlkEthOH_parmAtFrosst.offxml')
 ```
 Detailed examples of using SMIRNOFF with the toolkit can be found [in the documentation](https://open-forcefield-toolkit.readthedocs.io/en/latest/examples.html).
 
-## Chemical environments: Tools for chemical environment perception and manipulation
+## Chemical environments: Chemical environment perception and manipulation
```

Comment:
I think this is fine remaining here for now, but I don't think we're planning on maintaining this functionality for much longer. Maybe we can make a decision about this during the meeting tomorrow.